assets/css/main.css

```diff
@@ -80,6 +80,11 @@
   margin-top: 100px
 }
 
+.people-img {
```

Comment:
Putting this on the images themselves would likely fix the issue, but they're all different aspect ratios, so you'd have to set only the width or height to keep the image from getting distorted.